source/georef/georef.h

```diff
@@ -226,8 +226,14 @@ struct GeoRef {
         autocomplete::Autocomplete<unsigned int>(navitia::type::Type_e::POI);
 
     /// Indexe tous les nœuds
+
+    // TODO: Remove this before the next re-binarisation
```

Comment:
Is this comment still relevant ?

Reply:
yes! the old proximity_list includes the nodes of all 3 graphs, it was binarised... now it's split into three and they're rebuilt on the fly.. So no longer need to binarise.. but if i remove them from `serialize()`, we have to binarise...

Reply:
the old object is now removed ... comment removed...